frontend/libs/studio-components/src/components/StudioTextField/StudioTextfieldToggleView/StudioTextfieldToggleView.module.css

```diff
@@ -0,0 +1,10 @@
+.viewModeIconsContainer {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  color: var(--fds-semantic-text-neutral-default);
+}
+
+.pencilIcon {
+  margin-left: auto;
```

Comment:
Do we need this CSS? 🤔 😊

Reply:
Yes, based on Figma we need to have different colores and a space between the icons. maybe we can `remove justify-content: space-between ` because we have auto for `margin-left`